sdk/mongocluster/arm-mongocluster/src/api/firewallRules/index.ts

```diff
@@ -301,16 +273,15 @@ export function _firewallRulesListByMongoClusterSend(
 }
 
 export async function _firewallRulesListByMongoClusterDeserialize(
-  result:
-    | FirewallRulesListByMongoCluster200Response
-    | FirewallRulesListByMongoClusterDefaultResponse,
+  result: PathUncheckedResponse,
 ): Promise<_FirewallRuleListResult> {
-  if (isUnexpected(result)) {
+  const expectedStatuses = ["200"];
+  if (!expectedStatuses.includes(result.status)) {
     throw createRestError(result);
   }
 
   return {
-    value: result.body["value"].map((p) => {
+    value: result.body["value"].map((p: any) => {
```

Comment:
share the tsp definition for value type here.

Reply:
not sure if it's because
![image](https://github.com/user-attachments/assets/799662cc-973c-49ab-a2ac-662235b855d9)